noxfile.py

```diff
@@ -97,7 +97,7 @@ def default(session, django_dep=('django',)):
     )
 
 
-@nox.session(python=['2.7', '3.5', '3.6', '3.7'])
```

Comment:
Dropping Python 2.7 support is *definitely* a breaking change.

Reply:
Sorry, still getting used to the conventional commit style. Yes, this PR introduces a breaking change ([more discussion](https://github.com/googleapis/python-logging/pull/62) on the release PR). How do you think we should proceed since it was already merged without the breaking label?

Reply:
Given that we haven't removed the `Python :: 2` entry from `setup.py` yet, we can repair the breakage by doing that in another PR.